runtime/doc/lua.txt

```diff
@@ -2848,6 +2848,10 @@ vim.filetype.match({args})                              *vim.filetype.match()*
                 • {contents}? (`string[]`) An array of lines representing file
                   contents to use for matching. Can be used with {filename}.
                   Mutually exclusive with {buf}.
+                • {fast}? (`boolean`) Whether to skip time consuming detection
+                  steps, like pattern and content matching. Makes detection
+                  significantly faster at the cost of slightly less filetypes
```

Comment:
"fewer" instead of "less" for countable things (I think, @clason confirm pls)

Reply:
Yes, basically: https://www.merriam-webster.com/grammar/fewer-vs-less.

Reply:
I trust clason more than the dictionary

Reply:
> I trust clason more than the dictionary

Sure. My point was that your instinct was right. The explanation on that link explains that in general you should use *fewer* for things you can count (e.g., filetypes) and *less* things you measure (e.g., time), but that for *less* for *fewer* "does occur frequently in speech" (hence the common mistake). It also explains that there are several exceptions to the general rule, which is a pain (like many things in English).

Reply:
Thanks! Updated the wording.